nixos/release.nix

```diff
@@ -255,6 +255,15 @@ rec {
     }
   );
 
+  iso_cosmic = forAllSystems (
+    system:
+    makeIso {
+      module = ./modules/installer/cd-dvd/installation-cd-graphical-calamares-cosmic.nix;
+      type = "graphical";
+      inherit system;
+    }
+  );
+
```

Comment:
We shouldn't have anything in `release.nix` for it, so that Hydra doesn't build anything new. The manual already describes how to use the profiles in `modules/installer/cd-dvd/` to build ISOs manually, so I think simply having the COSMIC module in there is sufficient to enable users to build their own COSMIC ISOs.